src/Raven.Client/Json/Serialization/NewtonsoftJson/Internal/BlittableJsonWriter.cs

```diff
@@ -359,7 +359,10 @@ public override void WriteNull()
 
         public override void WriteValue(string value)
         {
-            _manualBlittableJsonDocumentBuilder.WriteValue(value);
+            if (value != null)
+                _manualBlittableJsonDocumentBuilder.WriteValue(value);
+            else
+                _manualBlittableJsonDocumentBuilder.WriteValueNull();
```

Comment:
what about6 all the other methods here that can get a null?

Reply:
All other types are already handled

Reply:
`LazyNumberValue` is a class, `LazyNumberValue`, the same, etc.